springboot/src/main/resources/application.properties

```diff
@@ -1 +1 @@
-
+spring.cloud.function.scan.enabled=false
```

Comment:
Spring Cloud Function by default scans for functions in `functions` package - even classes that have no Spring annotations. Disabling it potentially saves a little bit of the startup time.

Alternatively, all function beans could be moved to `functions` package and `@Component` annotation could be removed.